src/pages/components/breadcrumb/usage.mdx

```diff
@@ -49,30 +79,102 @@ The breadcrumb is a secondary navigation pattern that helps a user understand th
-| _Location-based_ | These illustrate the site's heirarchy and show the user where they are within that heirarchy.                                                                                              |
-| _Path-based_     | These show the actual steps the user took to get to the current page, rather than reflecting the site's information architecture. Path-based breadcrumbs are always dynamically generated. |
+### Main elements
+
+#### Page link
+
+- Each page link should be short and clearly reflect the location or entity it links to.
+- List pages starting with the highest-level parent page and move deeper down the information architecture as the breadcrumb trail progresses.
+- By default, the current page is not listed in the breadcrumb trail.
+
+### Overflow content
+
+When space becomes limited, use an [overflow menu](/components/overflow-menu/usage) to truncate the breadcrumbs. The first and last two page links should be shown, but the remaining breadcrumbs in between are condensed into an overflow menu. Breadcrumbs should never wrap onto a second line.
+
+<Row>
+<Column colLg={8}>
+
+![Truncated breadcrumbs](images/breadcrumb-truncated.png)
+
+</Column>
+</Row>
+
+### Further guidance
+
+For further content guidance, see Carbon's [content guidelines](/guidelines/content/general).
+
+## Behaviors
+
+### Interactions
+
+All the page links in the breadcrumb component should be interactive and link to their respective pages.
+
+#### Mouse
+
+Users can trigger an item by clicking on a breadcrumb page link. The separators between page links are not interactive.
+
+#### Keyboard
+
+Users can trigger a breadcrumb link by pressing `Enter` while the link has focus. For additional keyboard interactions, see the [accessibility tab](https://www.carbondesignsystem.com/components/breadcrumb/accessibility).
+
+Users can navigate between breadcrumb links by pressing `Tab` and `Shift-Tab`.
+
+#### Screen readers
+
+VoiceOver: Users can trigger a breadcrumb link by pressing `Enter` or `Control-Option-Space`. Users can navigate between breadcrumb links by pressing `Tab` and `Shift-Tab`.
+
+Jaws: Users can trigger a breadcrumb link by pressing `Enter`. Users can navigate between breadcrumb links by pressing the `Up` and `Down` arrow keys.
+
+NVDA: Users can trigger a breadcrumb link by pressing `Enter`. Users can navigate between breadcrumb links by pressing the `Up` and `Down` arrow keys.
+
+For additional information, see [screen reader tests](https://www.carbondesignsystem.com/components/breadcrumb/accessibility#macos-screen-reader-tests).
+
+## Modifiers
+
+By default Carbon breadcrumb trails should not include the current page. If a page doesn't have a title or the current page is not clear, it can be included in the breadcrumb trail. If the current page is included in a breadcrumb trail, it is always the last page link listed and is not an interactive link.
```

Comment:
```suggestion
By default, Carbon breadcrumb trails should not include the current page. If a page doesn't have a title or the current page is not clear, it can be included in the breadcrumb trail. If the current page is included in a breadcrumb trail, it is always the last page link listed and is not an interactive link.
```